content/news/2023-01-20-accessibility-report/index.md

```diff
@@ -0,0 +1,114 @@
+
+![Assortment of Tags on a History](./TagsOnHistoryDetails.png)
+
+*The new tag component*
+
+### Color
+
+![Tags on a History Item with different colors](./TagColors.png)
+
+Like the old tagging component, the tags color is determined randomly based on the tags name, so matching tags can easily be spotted.
+Previously HSL was used for this, by choosing a random Hue, and a random lightness within a range.
+
+HSL however has an issue: The lightness is not consistent across the hue spectrum. This means some colors appeared lighter than intended, others darker, making some tags hard to read.
+
+| ![2D HSL color spectrum](./hsl.png) | ![The lightness of HSL shown in a black and white image](./hsl-lightness.png) |
+|---|---|
+| HSL Color spectrum, mapping hue to saturation with fixed lightness | The same image showing just the perceived lightness value |
+*image sources: [HSLuv Comparison Page](https://www.hsluv.org/comparison/)*
+
+For this reason we switched to a new color space: HSLuv.
+This color space offers a uniform brightness across the hue/saturation spectrum, making the brightness of generated colors more predictable.
+This also allows us to make more precise adjustments to the color generation in the future.
+
+[Read more about HSLuv](https://www.hsluv.org/)
+
+### Contrast
+
+![Single tag with outline named "#named"](./TagOutline.png)
+
+The tags outline is back! Tags can be hard to tell apart from the background, when they have a uniform color.
+For this reason the new tag component reintroduces a subtle outline.
+The outline does not fully surround the tag, to give it a more modern appearance, while still offering the benefits of improved contrast.
+
+### Autocomplete
+
+![The tags autocompletion menu open, showing some options. Mouse hovering over option "ATACseq", showing a plus icon to the right of the option](./TagAutocomplete.png)
+
+The new Tags have an autocompletion menu.
+This menu can be easily navigated by keyboard or mouse, to add or remove tags.
+It is also the primary way of deleting tags via keyboard.
+Just search for a previously added tag and press enter to delete it.
+
+The auto-completion contains every tag in use on your galaxy account.
+
+## Keyboard Navigation in the History
+
+![History items with keyboard highlight](./HistoryKeyboard.png)
+
+Not just the tag component in the history got some keyboard upgrades, but also the history itself.
+Every button on the history items is now keyboard focusable, and the items can be expanded via keyboard.
+
+The keyboard focus also only steps into a history item if it is expanded, making it much faster to select the right item from the history. 
+
+## Heading Hierarchy, Aria Labels and Screen-Reader Texts
+
+A lot of mark up improvements have been done for screen readers.
+Heading hierarchies have been properly cleaned up, titles have been added to most sub-pages, and many elements have received aria labels.
```

Comment:
here maybe do you want to expand on why this is important? (even a link is probably fine) but I'm guessing most folks won't have the context and it would be nice to remind people why we're doing these things and how it benefits folks using accessibility tools